composer.json

```diff
@@ -31,11 +33,14 @@
   },
   "require-dev": {
     "dama/doctrine-test-bundle": "^6.3",
+    "escapestudios/symfony2-coding-standard": "^3.11",
     "friendsofphp/php-cs-fixer": "^2.16",
+    "mheap/phpunit-github-actions-printer": "^1.2",
     "phpstan/phpstan-doctrine": "^0.12.8",
     "phpstan/phpstan-symfony": "^0.12.2",
     "phpunit/phpunit": "^7.5",
     "roave/security-advisories": "dev-master",
+    "squizlabs/php_codesniffer": "^3.5",
```

Comment:
if you add csniffer, can you add the related `test` and `fix-cs` commands in the Makefile ?

Reply:
sure